### PR TITLE
Update Bundling & Minification doc

### PR DESCRIPTION
**Pull request by scottaddie**
Fixes https://github.com/aspnet/Docs/issues/4881

[Internal Review Page](https://review.docs.microsoft.com/aspnet/core/client-side/bundling-and-minification?branch=pr-en-us-4891)

### CODE REVIEW

**Comment by guardrex** [comment]
BundlerMinifier runs without the VS light up feature if the tooling ref is included. I discovered this when I was learning how to make it work from the CLI running under my use of VSC. Idk if there's any interest in including that as part of the topic to surface info for the non-VS users out there, but I just thought I'd mention it just in case.
```xml
<ItemGroup>
  <!-- Other app packages -->
  <PackageReference Include="BuildBundlerMinifier" Version="2.5.357" />
</ItemGroup>

<ItemGroup>
  <!-- Other tool references -->
  <DotNetCliToolReference Include="BundlerMinifier.Core" Version="2.5.357" />
</ItemGroup>
```

**Comment by scottaddie** [comment]
@guardrex You read my mind. I've had a few customers ask about this, so I do plan to include it.

**Comment by scottaddie** [comment]
@Rick-Anderson @madskristensen Please review.

File: aspnetcore/client-side/bundling-and-minification.md [hunk 1]
```diff
@@ -1,229 +1,309 @@
...
 
-## Overview
+## What is bundling and minification?
 
-In ASP.NET Core apps, there are multiple options for bundling and minifying client-side resources. The core templates for MVC provide an out-of-the-box solution using a configuration file and BuildBundlerMinifier NuGet package. Third party tools, such as [Gulp](using-gulp.md) and [Grunt](using-grunt.md) are also available to accomplish the same tasks should your processes require additional workflow or complexities. By using design-time bundling and minification, the minified files are created prior to the application’s deployment. Bundling and minifying before deployment provides the advantage of reduced server load. However, it’s important to recognize that design-time bundling and minification increases build complexity and only works with static files.
+Bundling and minification are two distinct performance optimizations you can apply in a web app. Used together, bundling and minification improve performance by reducing the number of server requests and reducing the size of the requested static assets.
```
Comment (Rick-Anderson) [inline aspnetcore/client-side/bundling-and-minification.md]: >Used together, bundling and minification improve performance by reducing the number of server requests and reducing the size of the requested static assets.

Too long for MT. Probably OK.

**Comment by Rick-Anderson** [comment]
[Link](https://github.com/aspnet/Docs/pull/4891/commits/2e905e2af7824bd61751a6d2a994c100aeb96606) to my changes

**Comment by scottaddie** [comment]
@Rick-Anderson Thank you for taking a look! Those changes look good to me.

**Pull request merged**